client/pages/schedule.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
```

Comment:
This looks great! We will actually be using this page in particular for when we sync the google calendar, so it will show up at the same time.